test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -736,6 +739,7 @@ public void ErrorFunction_DoesNotBringDownHost()
                 Content = new StringContent(testData)
             };
             request.SetConfiguration(new HttpConfiguration());
+            request.Content.Headers.ContentType = new MediaTypeHeaderValue("text/plain");
```

Comment:
Can you add like 2 more content headers and verify them below to test the multiplicity? I.e. symmetric with https://github.com/Azure/azure-webjobs-sdk-script/blob/658d4d0c9dce9af940bd1a4fd8d5fb0f2fb85e90/src/WebJobs.Script/Binding/HttpBinding.cs#L290